styles/style_option.h

```diff
@@ -0,0 +1,32 @@
+#ifndef STYLES_STYLE_OPTION_H
+#define STYLES_STYLE_OPTION_H
+
+// Usage StyleOption<OPTION, COLORS...>
+// OPTION: a number
+// COLORS: a color or style
```

Comment:
Each color if of type COLOR

Reply:
s/if/is/

Reply:
I am not familiar with "s/if/is/"?

Reply:
It's syntax used in some unix commands, like ed, sed, vi.
It means "substitute if with is".

Reply:
ah, thanks.